apps/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -139,7 +139,10 @@ update_pwd(Username, Fun) ->
 
 
 -spec(lookup_user(binary()) -> [mqtt_admin()]).
-lookup_user(Username) when is_binary(Username) -> mnesia:dirty_read(mqtt_admin, Username).
+lookup_user(Username) when is_binary(Username) ->
+    Trans = fun() -> mnesia:read(mqtt_admin, Username) end,
+    {atomic ,Users} = ekka_mnesia:transaction(?DASHBOARD_SHARD, Trans),
```

Comment:
we should avoid creating a nested transaction.

sorry, it was not clear in my last comment, i meant: create another API, like `lookup_user_t` to perform a transactional read.

or maybe just call `mnesi:read(mqtt_admin, Username)` at line 94, and leave `lookup_user` unchanged.

Reply:
Sorry, I didn't even notice the `nested transaction` problem. Create user is not commonly used, So i think `dirty` operation is enough.